test/sql/test_compiler.py

```diff
@@ -529,6 +529,13 @@ def test_limit_fetch_interaction(self):
             "SELECT 1 OFFSET :param_1 ROWS FETCH FIRST :param_2 ROWS ONLY",
             checkparams={"param_1": 2, "param_2": 1},
         )
+    
```

Comment:
please revert all changes to compiler.py.  this file must not be modifed in this PR